diagnostic.md

```diff
@@ -6,38 +6,31 @@ Write your answers inside this file, where it's indicated by the comments.
     What command do we use to do that? Use the correct URL for your fork of this
     repository in your answer.
 
-    ```sh
-    <!-- Remove this comment and place your answer here. -->
-    ```
+    We clone it by typing:
+    git clone git@github.com:louarnos/git-basics-diagnostic.git
+
 
 2.  What do you do after cloning a repository, but before starting work?
 
-    <!-- Remove this comment and place your answer here. -->
+    We cd to the new git repo and branch it
 
 3.  What command do we use to create a new branch? Name this branch `response`
     in your answer.
 
-    ```sh
-    <!-- Remove this comment and place your answer here. -->
-    ```
+    git branch response
 
 4.  We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
-    ```sh
-    <!-- Remove this comment and place your answer here. -->
-    ```
+    git status
 
 5.  We want to prepare a change for a commit by adding a file to the staging
     area. What command do we use? Suppose the change is in the current working
     directory and named `diagnostic.md`.
 
-    ```sh
-    <!-- Remove this comment and place your answer here. -->
-    ```
+    git add diagnostic.md
 
 6.  Should you ever edit published history?
 
-    ```sh
-    <!-- Remove this comment and place your answer here. -->
-    ```
+    Yes. Sometimes you need to reset/revert/amended a commit you made in the
```

Comment:
no it's not a good idea to edit published history. reseting and reverting is not editing your history.